### DISCUSSION TITLE
Rnode settings

### DISCUSSION THREAD

**Discussion by ronnietucker** [markqvist/Reticulum#646] [Help & Questions]
What settings are we supposed to put in for Hardware > Rnode (frequency, bandwidth, TX, etc.)?

I've used some settings I saw in a YouTube video, but I've no idea if they're right.

**Bonus question:** if two nodes were using different values, would they still be able to announce/discover each other??

**Comment by liamcottle** [comment]
> What settings are we supposed to put in for Hardware > Rnode (frequency, bandwidth, TX, etc.)?

Anything that is legal for where you are. Generally, people are referring to the Meshtastic docs as they have a great list of presets you could use as reference, as well as a slot calculator for picking frequencies.

https://meshtastic.org/docs/overview/radio-settings/

I would suggest using something that will give you the fastest data rate while at the same time being able to cover the distance you want. At the moment, it's unlikely you will find someone in your area using RNode, but if you do, you could switch to settings they are using to be able to communicate.

I would also suggest avoiding the default frequencies used by Meshtastic in your area, to avoid interference.

Having an SDR is also useful for being able to _see_ what is happening on the spectrum, and if you're lucky you might find some frequencies that no one is using :)

> I've used some settings I saw in a YouTube video, but I've no idea if they're right.

I would advise against using random settings for frequencies you found on YouTube, as these may not be allowed in your country.

> Bonus question: if two nodes were using different values, would they still be able to announce/discover each other??

For RNodes to be able to communicate with each other, they must have the same LoRa settings. This means the `frequency`, `bandwidth` and `spreading factor` must be the same for both RNodes.

If the settings are not the same, the LoRa chip on the receiving side will not decode it, because it's listening for something else.

**Reply by ronnietucker** [reply]
But wouldn't that mean everyone could be on slightly different settings and thinking there's no one in their area when there might be loads but everyone is shooting in different directions?

Still trying to get my head round Reticulum when coming from Meshtastic.

Thanks!

**Reply by fotografm** [reply]
In the UK the settings recommended by Andy Kirby are
867500000 Hz  ie. 867.5 MHz
Transmit power 22 dBm
125 kHz bandwidth
9 spreading factor
5 coding rate

I don't accept responsibility for the legality of these settings. I believe this is what most people in UK are using. Where are you ?

**Comment by faragher** [comment]
> But wouldn't that mean everyone could be on slightly different settings and thinking there's no one in their area when there might be loads but everyone is shooting in different directions?

Yes. Reticulum isn't a single network, but a tool for making networks. It's less turning on a radio and more dialing a phone number or typing in a URL. Multiple systems can run in the same region, so it's important to establish local groups with well-published information. If that means a bunch of people decide to simply say "Okay, this is Country A, we'll all use the same settings," that's fine, but that's up to the end user, not the network stack itself.

It does make for a slightly less "plug and play" approach, but asking about a local network's settings is the same as asking for a phone number or using a search engine. People just need to say "I'm starting a network in this area, here's what I'm using," and publish said information somewhere it can be found. This discussion board is as good a place as any.

>In the UK the settings recommended by Andy Kirby are

Two notes: Not all RNodes support 22 dBm, and it may be overkill for short range. The LilyGO RNodes, for example, have a limit of 17 dB. 

Second: The transmit power and coding rate don't need to be the same between radios. The frequency, bandwidth, and spreading factor all need to be the same, but power and coding rate can be tailored for the situation; they won't cause any problems.

**Reply by markqvist** [reply]
Spot on.

As a rule of thumb, most RNodes with SX1262 and SX1268 chips support 22dBm. Most RNodes with SX1276 and SX1278 chips only support 17 dBm.

**Comment by fotografm** [comment]
I agree with faragher and markqvist. Thank you for augmenting my comment.

**Comment by LinuxinaBit** [comment]
There is also this wiki page for common RNode LoRa settings that already has quite a few contributions from others in the community. It might be worth checking out as well:
https://github.com/markqvist/Reticulum/wiki/Popular-RNode-Settings
